tasks/freeswitch.yml

```
@@ -119,7 +119,7 @@
 
     - name: manage ipv6 sip_profiles
       become: true
-      ansible.builtin.command: |
+      command: >
```

Replacing `ansible.builtin.command` with `command` to satisfy the check… ok.

But replacing `>` with `|` should result in something not wanted according to my understanding of yaml.

`|` preserves newlines and `>` folds them into spaces which is kinda what I would expect from a `mv A B` command.

https://yaml-multiline.info/

I would expect to keep the `|`

```suggestion
      command: |
```

---

Hi @EdJoPaTo,
this is exactly the reason, why I changed it to `>` :D
On the command line, you would separate arguments for `mv` with spaces. So the behavior of `>` better emulates this, in my understanding.
However, both versions do work.

---

Oh yeah, guess I confused myself, you are right :D